fragments/home/news.yml

```diff
@@ -1,6 +1,6 @@
 - img: /img/homepage/secva.jpg
   alt: Video of VA Secretary Robert Wilkie's video announcing the new VA.gov.
-  title: VA.Gov Revised by Veterans
+  title: Welcome to the New VA.gov!
```

Comment:
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14963